.gitignore

```diff
@@ -1,3 +1,29 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
Seems like a little too much for this repo - most are usually in `~/.gitignore`?

Reply:
This is mostly a standard golang gitignore we used all over the plugin org